src/main/webapp/app/components/sidebar/OncoKBSidebar.tsx

```diff
@@ -80,18 +80,18 @@ const OncoKBSidebarExpanded = observer(
       const startX = mouseDownEvent.pageX;
 
       function onMouseMove(mouseMoveEvent: MouseEvent) {
-        const newWidth = oncoKBSidebarWidth + startX - mouseMoveEvent.pageX;
+        const newWidth = (oncoKBSidebarWidth ?? 0) + startX - mouseMoveEvent.pageX;
 
-        if (newWidth > oncoKBSidebarWidth && newWidth > closeNavWidth) {
-          closeNavigationSidebar();
+        if (newWidth > (oncoKBSidebarWidth ?? 0) && newWidth > closeNavWidth) {
+          closeNavigationSidebar?.();
         }
 
         if (newWidth <= ONCOKB_SIDEBAR_MIN_WIDTH) {
-          setOncoKBSidebarWidth(ONCOKB_SIDEBAR_MIN_WIDTH);
+          setOncoKBSidebarWidth?.(ONCOKB_SIDEBAR_MIN_WIDTH);
         } else if (newWidth >= maxWidth) {
-          setOncoKBSidebarWidth(maxWidth);
+          setOncoKBSidebarWidth?.(maxWidth);
         } else {
-          setOncoKBSidebarWidth(oncoKBSidebarWidth + startX - mouseMoveEvent.pageX);
+          setOncoKBSidebarWidth?.((oncoKBSidebarWidth ?? 0) + startX - mouseMoveEvent?.pageX);
```

Comment:
How come `mouseMoveEvent` needs optional chaining, but not on line 83? If it can really be undefined, then the function definition should probably reflect that?

Reply:
Good catch! It didn't need that.